test/fw/ptl/utils/pbs_testusers.py

```diff
@@ -209,3 +217,8 @@ def __int__(self):
 PBS_ROOT_USERS = (PBSROOT_USER, ROOT_USER)
 
 PBS_BUILD_USERS = (BUILD_USER,)
+
+PBS_TEST_USERS = (TEST_USER, TEST_USER1, TEST_USER2, TEST_USER3)
```

Comment:
Please rename this variable to something else like `REQUIRED_USERS` as these are not *only* test users, tests users are what we have defined in PBS_USERS, this list is a list which specifies a list of users which must exist in systems.

Also why we are checking the only a subset of PBS_USERS for existence? Shouldn't all user/group define in this file should exist in the system? By checking a subset of PBS_USERS for existence, we are saying that we don't need other users specified in PBS_USERS so if that's the case then we should remove those users from PTL but that is not the case...

Reply:
Was there any reason why we were checking initially only 4 TEST_USERS ? I have still kept this code as it is until we decide why it was there before and why it doesn't include all users.